kube-client/src/config/file_config.rs

```diff
@@ -328,6 +329,24 @@ impl Kubeconfig {
     }
 }
 
+impl FromStr for Kubeconfig {
+    type Err = KubeconfigError;
+
+    fn from_str(text: &str) -> Result<Self, Self::Err> {
+        Kubeconfig::from_text(text)
+    }
+}
+
+fn kubeconfig_from_text(text: &str) -> Result<Vec<Kubeconfig>, KubeconfigError> {
```

Comment:
```suggestion
fn kubeconfig_from_yaml_str(text: &str) -> Result<Vec<Kubeconfig>, KubeconfigError> {
```

Reply:
Hm, perhaps `kubeconfig_from_yaml` then? `...from_yaml_str` feels unnecessary verbose

Reply:
That runs into risk of confusion with importing from a YAML *file*. But sure, that works too.

Reply:
it's ultimately a private helper, so doesn't matter much.

Reply:
True.

Reply:
Very well